### DISCUSSION TITLE
Add "Day" on the Event Text Variable

### DISCUSSION THREAD

**Discussion by BabyLoves** [AscensionGameDev/Intersect-Engine#1486] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not.

**Describe the solution you'd like**
Actually are the Event Text Variables that show some informations about Time(hour,minute, second and period(AM/PM).
My suggestion is add the "Day"(week) in the Time Informations like "Sun, Mon, Tue, Wed, Thu, Fri, Sat" and maybe possible use it in the Events too

**Additional context**
Using \day \hour \minute \second //////      (\period)
Result: Sun - 14:20:16   /////      (or Sun 02:20:16 PM)

**Comment by Cheshire92** [comment]
Are these variables tied to in-game time or server real world time?

**Comment by BabyLoves** [comment]
> Are these variables tied to in-game time or server real world time?

Real time I guess...

**Comment by WeylonSantana** [comment]
Just to add information, this information above is related to the server time, so maybe that's why don't have a \day.

**Comment by WeylonSantana** [comment]
More information, it should be considered the local time
Here in the southern hemisphere we use DD/MM/YYYY, in the north MM/DD/YYYY, the location should be taken into account
